lib/matplotlib/backends/backend_svg.py

```diff
@@ -360,7 +360,8 @@ def _write_metadata(self, metadata):
                                  'Expected str, date, datetime, or iterable '
                                  'of the same, not {!r}.'.format(type(date)))
             metadata['Date'] = '/'.join(dates)
-        else:
+        elif 'Date' not in metadata:
```

Comment:
Can we do this as

```python
sentinel = object()
date = metadata.get('Date', sentinel)
if date is sentinel:
    # do default thing
else if date is not None:
    # do type annotation thing
else:
    # do nothing

Reply:
I've considered using `False` as the sentinel but I felt like that's too much a hack and is potentially user observable.

For creating a new object, according to my naive benchmark on cpython 3.8 it's actually more expensive then doing a lookup from a small dict

```python
In [1]: %timeit object()
42.4 ns ± 0.619 ns per loop (mean ± std. dev. of 7 runs, 10000000 loops each)

In [2]: a = {}

In [3]: %timeit 'a' in a
25.6 ns ± 0.493 ns per loop (mean ± std. dev. of 7 runs, 10000000 loops each)

In [4]: a = {'b': 'c', 'd': 'e'}

In [5]: %timeit 'a' in a
25.5 ns ± 0.327 ns per loop (mean ± std. dev. of 7 runs, 10000000 loops each)
```

so I'm not sure if it's worth doing = = ......

Reply:
My motivation is readability rather than performance.